### PR TITLE
update PROJECT file

### PR DESCRIPTION
**Pull request by iamniting**
Signed-off-by: Nitin Goyal <nigoyal@redhat.com>

### CODE REVIEW

File: PROJECT [hunk 1]
```diff
@@ -1,42 +1,51 @@
 domain: openshift.io
-layout: go.kubebuilder.io/v2
+layout:
+- go.kubebuilder.io/v3
```
Comment (malayparida2000) [inline PROJECT:3]: AFAIK kubebuilder.io/v3 also requires other changes in our project directory & other places.

File: PROJECT [hunk 2]
```diff
@@ -1,42 +1,51 @@
...
+plugins:
+  manifests.sdk.operatorframework.io/v2: {}
+  scorecard.sdk.operatorframework.io/v2: {}
 projectName: ocs-operator
-repo: github.com/red-hat-storage/ocs-operator
+repo: github.com/red-hat-storage/ocs-operator/v4
```
Comment (malayparida2000) [inline PROJECT:8]: I think the repo link won't contain the /v4, going to that url gives a 404 error.

**Review by malayparida2000 — CHANGES_REQUESTED** [review]

**Review by malayparida2000 — APPROVED** [review]
/lgtm

File: PROJECT (hunk 1 above)
Comment (iamniting) [reply PROJECT:3]: We sorted this over a call.

File: PROJECT (hunk 2 above)
Comment (iamniting) [reply PROJECT:8]: We sorted this over a call

**Pull request merged**